src/libs/runtime-utils.js

```diff
@@ -81,14 +88,27 @@ export const runtimeCost = ({ runtimeConfig, status }) => {
   }
 }
 
+/*
+ * - Disk cost is incurred regardless of app status.
+ * - Default nodepool VMs always run and incur compute and external IP cost whereas app
+ *   nodepool VMs incur compute and external IP cost only when an app is running.
+ * - Default nodepool cost is shared across all Kubernetes cluster users. It would
+ *   be complicated to calculate that shared cost dynamically. Therefore, we are being
+ *   conservative by adding default nodepool cost to all apps on a cluster.
+ * - Disk cost is for 250Gb of NFS disk, 10Gb of postgres disk, and 200Gb of boot disks (1 boot disk per nodepool)
+ */
 export const getGalaxyCost = app => {
-  // numNodes * price per node + diskCost + defaultNodepoolCost
-  const defaultNodepoolCost = machineCost('n1-standard-1')
-  const appCost = app.kubernetesRuntimeConfig.numNodes * machineCost(app.kubernetesRuntimeConfig.machineType) +
-    persistentDiskCost({ size: 250 + 10 + 100 + 100, status: 'Running' })
-  return appCost + defaultNodepoolCost
-  // diskCost: 250Gb for the NFS disk, 10Gb for the postgres disk, and 200Gb for boot disks (1 boot disk per nodepool)
-  // to do: retrieve the disk sizes from the app not just hardcode them
+  const appStatus = app ? app.status : 'UNKNOWN'
+  const defaultNodepoolComputeCost = machineCost('n1-standard-1')
+  const defaultNodepoolIpAddressCost = ephemeralExternalIpAddressCost(1, 0)
+
+  // TODO: Retrieve disk sizes from the app instead of hardcoding them
+  const diskCost = persistentDiskCost({ size: 250 + 10 + 100 + 100, status: 'Running' })
+
+  const staticCost = defaultNodepoolComputeCost + defaultNodepoolIpAddressCost + diskCost
+  const dynamicCost = app.kubernetesRuntimeConfig.numNodes * machineCost(app.kubernetesRuntimeConfig.machineType) + ephemeralExternalIpAddressCost(app.kubernetesRuntimeConfig.numNodes, 0)
+
+  return (['RUNNING', 'PROVISIONING', 'PRECREATING', 'STARTING'].includes(appStatus.toUpperCase())) ? (staticCost + dynamicCost) : staticCost
```

Comment:
Should the `STOPPING` case also display the whole cost (`staticCost + dynamicCost`) to be conservative? 🤔